src/index.html

```diff
@@ -11,7 +11,219 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-div-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+
+      <fieldset class="personalinfo">
```

Comment:
Don't add empty lines between children and parents
```suggestion
    >
      <fieldset class="personalinfo">
```